classes/submission/genre/Repository.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace PKP\submission\genre;
+
+use PKP\submission\genre\Genre;
+use Illuminate\Database\Eloquent\Collection;
+use PKP\db\XMLDAO;
+
+
+class Repository
+{
+
+    /**
+     * Get enabled genres by context ID.
+     *
+     * @param int $contextId
```

Comment:
The `@param` phpdocs don't add anything that's not already in the function declaration, so please remove them. (Here and below.)